packaging/debian/control

```diff
@@ -0,0 +1,25 @@
+Source: cockpit-navigator
+Section: admin
+Priority: optional
+Maintainer: Utopia Maintenance Team <pkg-utopia-maintainers@lists.alioth.debian.org>
+Uploaders: Martin Pitt <mpitt@debian.org>,
+Build-Depends: debhelper-compat (= 13),
+               gettext (>= 0.19.7),
+               gettext (>= 0.21) | appstream,
+Standards-Version: 4.6.2
+Rules-Requires-Root: no
+Homepage: https://github.com/cockpit-project/cockpit-navigator
+Vcs-Git: https://salsa.debian.org/debian/cockpit-navigator.git
+Vcs-Browser: https://salsa.debian.org/debian/cockpit-navigator
```

Comment:
These don't exist yet, but indeed that's where they will be, so let's keep them in.